source/ways-of-working/new-component/connect-via-vpn.html.md.erb

```diff
@@ -29,5 +29,5 @@ Once this is setup and deployed, you can then make a request to PlatOps in \#pla
 ```` 
```

Comment:
Can platops-help be linked to https://hmcts-reform.slack.com/app_redirect?channel=platops-help